### PR TITLE
Problem: crossfire config not possible (fixes #350)

### LINKED ISSUE #350: Problem: crossfire config not possible
The crossfire network should use the "mainnet" parameters (HD paths, bech32 prefixes...), but "tcro" denomination to denote it's a test token

### PR DESCRIPTION
**Pull request by tomtau**
Solution: added another network build configuration option

### CODE REVIEW

**Comment by tomtau** [comment]
just for the context, as mainly @calvinaco and @lezzokafka may be familiar with it -- crossfire will use the mainnet parameters (bech32 prefixes, HD path...), but should use the "test cro" token denomination

**Comment by yihuang** [comment]
> just for the context, as mainly @calvinaco and @lezzokafka may be familiar with it -- crossfire will use the mainnet parameters (bech32 prefixes, HD path...), but should use the "test cro" token denomination

maybe it's more straightforward to make a new network type?

**Comment by tomtau** [comment]
> > just for the context, as mainly @calvinaco and @lezzokafka may be familiar with it -- crossfire will use the mainnet parameters (bech32 prefixes, HD path...), but should use the "test cro" token denomination
> 
> maybe it's more straightforward to make a new network type?

yeah, perhaps -- I see the Cosmos SDK only has one "baseDenom", so currency conversions won't work for tcro

**Review by yihuang — APPROVED** [review]

File: config/prefix_crossfire.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// +build testnet,crossfire
```
Comment (devashishdxt) [inline config/prefix_crossfire.go:1]: Shouldn't this be `!testnet`?

**Pull request merged**